packages/react-ui/accessibility/src/FocusGrid.js

```diff
@@ -7,11 +7,11 @@
  * @flow
  */
 
-import type {KeyboardEvent} from 'react-events/src/dom/Keyboard';
+import type {KeyboardEvent} from '../../events/src/dom/Keyboard';
```

Comment:
Maybe these paths should also be `react-ui/events...` as that makes it easier to avoid inlining dependencies in the bundles